### PR TITLE
Makes mm-datepicker "close" link and "save" button text user definable

### PR DESCRIPTION
**Pull request by jeremysklarsky**
Updated Documentation

### CODE REVIEW

File: src/mm-datepicker/mm-datepicker.js [hunk 1]
```diff
@@ -58,6 +58,9 @@
 			state:'closed',
 			valign: { value: "center", reflect: true },
 			align: { value: "center", reflect: true },
+
+			closeText: "Close",
```
Comment (dlasky) [inline src/mm-datepicker/mm-datepicker.js]: super nitpicky, but could you change these to be closeLabel, saveLabel?

other than that +1

**Comment by dlasky** [comment]
+1 pending var naming

**Comment by jeremysklarsky** [comment]
yes no problem - my only question was about the var names, i even thought about using label!

**Comment by derekolson** [comment]
+1

**Pull request merged**